recipes/diffpy.srreal/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "diffpy.srreal" %}
+{% set version = "v1.3.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/diffpy/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: d0e4e847e39689a6d552ed64579b3e1f6f8cfee31130f43f54e4395935f0947e
+
+build:
+  # we need egg_dir to avoid overlapping diffpy/__init__.py files.
+  preserve_egg_dir: true
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
+    - {{ compiler('cxx') }}
+  host:
+    - python {{ python }}
+    - setuptools
+    - scons
+    - pip
+    - numpy {{ numpy }}
+    - libdiffpy
+    - libboost-python-devel
+
+  run:
+    - libdiffpy # NOTE libdiffpy is added implicitly from libdiffpy run_exports
+    - python
+    - setuptools
+    - {{ pin_compatible('numpy', min_pin='x.x', max_pin='x') }}
+    - libboost-python-devel
```

Comment:
Please don't depend on devel packages in run. The run_exports should be sufficient.
```suggestion
```